lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -338,8 +340,11 @@ private Hover collectProperty(IPositionRequest request, String property) {
 	// TODO: Move this function to a utility class
 	public static Map<String, String> getMavenProjectProperties(MavenProject project) {
 		Map<String, String> allProps = new HashMap<>();
+		
+		Properties projectProperties = project.getProperties();
+		EnvironmentUtils.addEnvVars(projectProperties);
```

Comment:
`EnvironmentUtils.addEnvVars(projectProperties);` should be refactored & called in the constructor or somewhere else, so that the result is cached.

Reply:
Still needs to be done.